dockers/docker-base/Dockerfile.j2

```diff
@@ -25,12 +25,14 @@ RUN apt-get clean && apt-get update
 RUN apt-get -y install \
     rsyslog            \
     supervisor         \
-    vim-tiny           \
+    vim                \
```

Comment:
With the help of layer reuse and docker squash (https://docs.docker.com/engine/reference/commandline/build/), the total disk size of the docker images and containers in a production switches is about 570M. Currently we are still deploying images for small disk switches.